packages/lit-dev-content/site/blog/2023-09-27-lit-3.0-prerelease-2.md

```diff
@@ -0,0 +1,133 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Lit 3.0 Prerelease 2 and more!"
+summary: "Today we are publishing a second of prereleases for the upcoming Lit 3.0 launch."
+date: 2023-09-27
+author:
+  - justin-fagnani
+---
+
+# Lit 3.0 Prerelease 2 and more!
+
+Today we are publishing a second of prereleases for the upcoming Lit 3.0 launch. This includes the core packages of `lit`, `lit-html`, and `lit-element`, prereleases for the Task, Context, and React packages that are graduating from Lit Labs, and previews of two exciting new Lit Labs packages: `@lit-labs/compiler` and `@lit-labs/preact-signals`.
+
+The full list of prerelease packages is:
+- `lit@3.0.0-pre.1`
+- `lit-html@3.0.0-pre.1`
+- `lit-element@4.0.0-pre.1`
+- `@lit/reactive-element@3.0.0-pre.1`
+- `@lit/context@1.0.0-pre.1`
+- `@lit/task@1.0.0-pre.1`
+- `@lit/react@1.0.0-pre.1`
+- `@lit/task@1.0.0-pre.1`
+- `@lit-labs/compiler@0.1.0-pre.1`
+- `@lit-labs/preact-signals@0.1.0-pre.1`
+
+## Lit 3.0 Prerelease 2
+
+As we [announced with the first prerelease](./2023-05-15-lit-3.0-prerelease/), Lit 3.0 is a breaking change that most notably removes support for IE11. Lit 3.0 is intended to have no new features, since new features can be added in non-breaking minor versions.
+
+### New hybrid decorators with standard decorators support
+
+One new thing we _have_ added to Lit 3.0 though is support for TC39 standard decorators. The new decorator spec has reached Stage 3 in TC39, meaning that browsers and compilers are now implementing them. This is a huge step for Lit because it allows us to begin the process of moving to decorator implementation that won't require a compiler to use.
```

Comment:
```suggestion
One new thing we _have_ added to Lit 3.0 though is support for TC39 standard decorators. The new decorator spec has reached Stage 3 in TC39, meaning that browsers and compilers are now implementing them. This is a huge step for Lit because it allows us to begin the process of moving to a decorator implementation that won't require a compiler to use.
```